checkov/arm/checks/resource/MSSQLServerMinTLSVersion.py

```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+
+
+class MSSQLServerMinTLSVersion(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure MSSQL is using the latest version of TLS encryption"
+        id = "CKV_AZURE_52"
```

Comment:
This check has already been implemented here - 
https://github.com/bridgecrewio/checkov/pull/6245